__test__/groups.test.ts

```diff
@@ -55,7 +54,7 @@ describe("loadGroupsFromOnestopIntoScriptProperties", () => {
 
         const setScriptPropertyMock: Mock = jest.fn();
         jest.mock("../src/main/propertiesService", () => ({
-            loadMapFromScriptProperties: jest.fn(),
```

Comment:
Because this function is getting mocked with `jest.fn()` which returns undefined, the following line was crashing because `GROUPS_MAP` was undefined
https://github.com/andrewc5716/onestop-basecamp-integration/blob/329b317aaf082b64c92d92153c1abbecac4dafe8/src/main/groups.ts#L29